Source/WebCore/platform/graphics/FontCascade.h

```diff
@@ -150,12 +151,14 @@ class FontCascade : public CanMakeWeakPtr<FontCascade> {
 
     bool isSmallCaps() const { return m_fontDescription.variantCaps() == FontVariantCaps::Small; }
 
-    float wordSpacing() const { return m_wordSpacing; }
-    float letterSpacing() const { return m_letterSpacing; }
+    float letterSpacing() const;
+    float wordSpacing() const;
+    const Length& computedLetterSpacing() const { return m_spacing.letter; }
+    const Length& computedWordSpacing() const { return m_spacing.word; }
```

Comment:
I would expect absolute pixel lengths with the name `computedLetterSpacing`, which I don't think we're necessarily returning here (it sounds like we might preserve percentages in here).